slave/download.py

```diff
@@ -43,11 +43,11 @@ def download_from_url(url):
 
     raise Exception("Unknown retriever")
 
-def download_for_repo(config, repo, cset="latest"):
+def download_for_repo(config, repo, cset="latest", buildtype='opt'):
```

Comment:
Do we need to specify a default value for `buildtype`, since it's explicitly passed from the callers?